packages/generator-emakinacee-react/generators/app/templates/static/config/rewireWorkboxGenerate.js

```diff
@@ -0,0 +1,24 @@
+const { rewireWorkboxGenerate } = require('react-app-rewire-workbox');
+
+module.exports = function override(config, env) {
+    if (env !== "production") return config;
```

Comment:
Does this mean that service workers are not available during development? Is that because of caching issues?

Reply:
Yes, you are right.
During testing and using it in development I always had some trouble with cache, HMR etc.

But for local testing you can just `npm run build` and then try it out with `npm run serve:build` before deployments.  
I will also add this hint to the readme part about workbox/service workers